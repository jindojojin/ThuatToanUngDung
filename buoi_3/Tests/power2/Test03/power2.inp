12
1000000 1000000 1000000 1000000 1000000 1000000 1000000 1000000 1000000 1 1048576 1024
