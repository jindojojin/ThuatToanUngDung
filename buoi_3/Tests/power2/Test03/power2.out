1 1024 1048576
